request-node/values.yaml

```
@@ -67,6 +68,8 @@ ipfs:
     port: 4001
   # IPFS API port
   api:
+    # required if `ipfs.sidecar` is false
+    # host: 
```

Would this work? It wasn't obvious to me at first glance that the comment above was applying to the "host" property, since host appears greyed out, I thought the comment was applying to "port".
```suggestion
    host: null
```

---

better?

---

Perfect  👌